data/json/monsters/nether.json

```diff
@@ -1074,8 +1074,8 @@
   {
     "id": "mon_memory",
     "type": "MONSTER",
-    "name": { "str": "Memory", "str_pl": "Memories" },
-    "description": "The shade of a human body reaching towards you like it needs help.  Though it has no eyes you feel as though it is staring through you.",
+    "name": { "str": "memory", "str_pl": "memories" },
+    "description": "The silhouette of a human body, reaching towards you like it needs help.  Though it has no eyes, you feel an intense sense of being watched.",
```

Comment:
Being intensely watched is the opposite of something staring through you. 

I'm indifferent to it being changed but it is the exact opposite feeling. They aren't attacking *you* they are somehow recreating the fear of their last moments so they are looking at something past you. Your just getting caught in the memory.